languages/swift/exercises/concept/strings-and-characters/.docs/after.md

```diff
@@ -0,0 +1,188 @@
+var empty = ""
+```
+
+Character literals in Swift look just like string literals, however, only one character is allowed between the quotation marks. Since the two literals look the same, Swift will default to inferring that characters are strings unless the type is specified by an explicit annotation or the context can tell the compiler otherwise.
+
+```swift
+let aString = "A"
+// Swift infers this to be a String
+
+var aChar: Character = "A"
+// Swift now knows to make this a Character
+
+let badChar: Character = "Too many characters"
+// Error: Cannot convert value of type 'String' to specified type 'Character'
+```
+
+Like other constructs in Swift, characters and strings are immutable if defined using `let` and mutable if defined using `var`. So in the above examples, `hello` and `sausage` are immutable strings while `fam` and `empty`, AND `aChar` are mutable.
+
+### Type conversion
+
+Type conversion allows us to get the string representation of many different types in Swift, including `Int`s, `Double`s and `Character`s.
+
+```swift
+String(110)
+// => "110"
+String(Double.pi)
+// => "3.141592653589793"
+String("x" as Character)
+// "x"
+```
+
+Some of the conversions, like those for `Int` allow for different equivalent representations:
+
+```swift
+String(113, radix: 2)
+// => "1110001"
+String(3735928559, radix: 16)
+// => "deadbeef"
+```
+
+### Building strings from smaller parts
+
+A character or string can be appended to a string through the string's `append(_:)` method:
+
+```swift
+var greeting = "Hello"
+let world = ", world"
+let period: Character = "."
+greeting.append(world)
+// greeting is now: "Hello, world"
+greeting.append(period)
+// greeting is now: "Hello, world."
+```
+
+Strings can be concatenated using the `+` operator, and mutable strings can have other strings appended onto them using the `+=` mutating operator.
+
+```swift
+"honey" + "comb"
+// => "honeycomb"
+
+fam += ", this is our cat: 🐱"
+// => "This is my family: 👨‍👩‍👦‍👦, this is our dog: 🐶, this is our cat: 🐱"
+```
+
+Though this can have unexpected effects when Unicode characters are in play.
+
+```swift
+let zeroWidthJoiner = "\u{200D}"
+"🏳️" + zeroWidthJoiner + "🌈"
+ // => "🏳️‍🌈"
+```
+
+Note that only two strings can be combined this way. In order to use `+` with a Character, it must first be converted to a String:
+
+```swift
+let question: Character = "?"
+let areYouHappy = "Happy" + question
+// Error: Cannot convert value of type 'Character' to expected argument type 'String'
+
+let areYouHappy = "Happy" + String(question)
+// => "Happy?"
+```
+
+Strings may also be built up out of values of other types in two other ways: format strings, or string interpolation.
+
+### Format strings
+
+Format strings are similar to those seen in other languages, where a [string with special tokens inside][string-format-specifiers] is followed by a list of components that are inserted into the string at the location of the tokens, formatted as directed by the tokens:
```

Comment:
Just a FYI: some tracks have a separate exercise for working with string using more complex formatting stuff.

Reply:
I may add an exercise like that, though in Swift they aren't used much. You're more likely to see some combo of string interpolation, number formatters, date formatters, etc. Though each of those can use an exercise down the road.

Reply:
Okay great